.github/build_all_examples.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+# BSP dictionary. README strings -> component names
+declare -A bsp_dictionary
+bsp_dictionary['Wrover kit']='esp_wrover_kit'
```

Comment:
This might deserve a comment, "when adding new boards here, don't forget to add them to examples/bsp_ext.py as well", and a similar comment in bsp_ext.py.